blazor/predefined-dialogs/position.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Positioning in Blazor Predefined Dialog Component | Syncfusion
+description: Checkout and learn here all about positioning in Syncfusion Predefined Dialogs component and much more details.
+platform: Blazor
```

Comment:
Platform: Blazor

Reply:
In `Breadcrumb` component they have given lower case only. So not corrected this changes.

<img width="662" alt="image" src="https://user-images.githubusercontent.com/93525217/182564491-6280ce19-93a0-4997-94d2-e4708dae4964.png">